engine/spec/components/citizens_advice_components/header_spec.rb

```diff
@@ -6,39 +6,41 @@
       render_inline(described_class.new)
     end
 
-    it "does not render without any slots" do
+    specify "does not render without any slots" do
```

Comment:
If these are aliases, shall we just use `it`?   It (lol) certainly reads better here